package.json

```diff
@@ -7,7 +7,7 @@
     "viv": "VIV_PORT=3000 node --loader ts-node/esm src/app.ts",
     "lint": "eslint src static",
     "lint-markdown": "markdownlint-cli2 --config .github/.markdownlint-cli2.yaml",
-    "test": "node --loader ts-node/esm tests/unit/cli.ts",
+    "test": "node --loader ts-node/esm --test 'tests/unit/*'",
```

Comment:
CI fails with this:

```
Run yarn test
yarn run v1.22.22
$ node --loader ts-node/esm --test 'tests/unit/*'
Could not find '/home/runner/work/Vivify/Vivify/tests/unit/*'
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Error: Process completed with exit code 1.
```

Reply:
Single quotes to double quotes didn't help

Reply:
Giving two explicit paths like `--test tests/unit/cli.ts --tests/unit/alerts.ts` at least reveals a bit about the problem:

```
Could not find '/home/runner/work/Vivify/Vivify/--test'
```

Reply:
Ok it can now run the tests but fails on one of them, let's see.